app/controllers/subscriber_lists_controller.rb

```diff
@@ -31,6 +31,25 @@ def create
     render json: subscriber_list.to_json
   end
 
+  def update
```

Comment:
You could avoid needing to specify the `:title` in three places (twice on line 49 and once on line 77) by taking a hash of updates, like so:

```ruby
def update
  updates = params.permit(updateable_parameters).to_h # the `to_h` might not be necessary - check that

  if updates.empty?
    # do the unprocessable entity here
  end

  # etc

  subscriber_list.update!(updates)

  # etc
end
```

Reply:
Also, do you want to protect against nulls?  eg, PATCHing `{"title":null}` feels like it shouldn't blank out the title to me.  In which case your current code doesn't handle that.  In my suggestion you can add `.compact` onto the `updates` line.

Reply:
yeah you are right, permit is the right tool here, I'm doing something non-standard